airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/avro/JsonToAvroSchemaConverter.java

```diff
@@ -266,4 +270,12 @@ Schema getNullableFieldTypes(final String fieldName, final JsonNode fieldDefinit
     }
   }
 
+  private static JsonNode getTypeStringSchema() {
+    return Jsons.deserialize("{\n"
+        + "    \"type\": [\n"
+        + "      \"string\"\n"
+        + "    ]\n"
+        + "  }");
+  }
```

Comment:
You can reuse the `STRING_SCHEMA` constant, and setup the Avro schema directly.

Reply:
fixed